src/index.html

```diff
@@ -22,6 +22,408 @@
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="MOYO"
+          class="logo_img"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_ul">
+          <li class="nav_link">
+            <a
+              href="index.html"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_link">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_link">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav_link"
+            data-qa="nav-hover"
+          >
+            <a
+              href="index.html"
+              class="link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_link">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav_link">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav_link">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav_link">
+            <a
+              href="index.html"
+              class="link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__photo"
+          src="./images/imac.jpeg"
+          alt="Monoblock Apple"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <h3 class="card__product-code">Product code: 195434</h3>
+
+        <div class="rating">
+          <div class="stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+
+          <h3 class="rating__reviews">Reviews: 5</h3>
+        </div>
+
+        <div class="price">
+          <h3 class="price__text">Price:</h3>
+          <p class="price__numbers">&#36;2,199</p>
+        </div>
+
+        <a
+          href="index.html"
+          class="button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```